client/gutenboarding/gutenboard.tsx

```diff
@@ -47,7 +47,7 @@ export function Gutenboard() {
 			/>
 			<SlotFillProvider>
 				<DropZoneProvider>
-					<BlockEditorProvider value={ [ onboardingBlock ] }>
+					<BlockEditorProvider value={ [ onboardingBlock ] } settings={ { templateLock: 'all' } }>
```

Comment:
This isn't typed as a valid setting, I asked in core slack for guidance:

https://wordpress.slack.com/archives/C5UNMSU4R/p1571825488172300